autoprotocol/protocol.py

```diff
@@ -5507,7 +5506,7 @@ def image(self, ref, mode, dataref, num_images=1, backlighting=None,
             raise TypeError("num_images must be a positive integer.")
         if magnification:
             if not isinstance(magnification, (float, int)) or \
-               magnification <= 0:
+                magnification <= 0:
```

Comment:
nit: don't refactor code out of scope of PR

Reply:
undone